css/styles.css

```diff
@@ -597,6 +597,44 @@ input.submit[type=reset] {
    background-color:#eeeeee;
 }
 
+#kb_browse {
+   display: flex;
+   align-items: stretch;
+   border: 1px solid #E6E6E6;
+   height: 100%;
+   min-height: 60vh;
+   margin-left: -10px;
+   margin-top: -5px;
+}
+
+#kb_browse .kb_tree {
+   min-width: 250px;
+   padding-right: 10px;
+   text-align: left;
+   background-color: #F8F8F8;
+}
+
+#kb_browse .kb_tree a {
+   font-weight: normal;
+}
+
+#kb_browse .kb_tree_search {
+   width: 100%;
+   margin: 5px;
+   padding: 10px 5px;
+   border-radius: 0;
+   box-sizing: border-box;
+}
+
+#kb_browse .kb_items {
+   padding: 10px;
```

Comment:
A `flex-grow:1;` would make item take all remaining space. I think it is better than having a blank space on left for wide screens.

Reply:
Since this is not yet used at all, I'm not going do do this change. Let's work on those consideration when changing UI.